### DISCUSSION TITLE
Using ngxtension to migrate towards "Modern" Angular

### DISCUSSION THREAD

**Discussion by giscus** [timdeschryver/timdeschryver.dev#337] [Announcements]
# Using ngxtension to migrate towards "Modern" Angular

Using ngxtension to migrate towards "Modern" Angular

https://timdeschryver.dev/bits/using-ngxtension-to-migrate-towards-modern-angular

**Comment by kishoreandra** [comment]
Amazing 🤩 Thanks Tim for writing one 🙏, ...

Q) Would there any problem using both DI and inject for different services 🤔 ?

✅ Less code and more readable for both input and output 🥳

**Reply by timdeschryver** [reply]
Thanks @kishoreandra ! 
As far as I know, that won't be a problem (I haven't experienced any).

**Reply by kishoreandra** [reply]
Good to know, thanks Tim
